com.unity.multiplayer.mlapi/Runtime/Configuration/NetworkPrefab.cs

```diff
@@ -34,12 +57,27 @@ internal uint Hash
                     return 0;
                 }
 
-                var networkObject = Prefab.GetComponent<NetworkObject>();
+                var prefabGameObject = Prefab;
+                
+                switch(Override)
```

Comment:
```suggestion
                switch (Override)
```

I'm the spaceman :P

Can we kill this `internal uint Hash { get { ... } }` property entirely?
It seems possible to me because we already have `NetworkPrefabOverrideLinks` now and there should be just a few places that use this getter property directly, no?
(and this is a minor/optional change suggestion, I'm just curious)

Reply:
We can't as it is used in one other place that currently requires it.
NetworkConfig.GetConfig in order to force prefabs to be the same.